src/main/java/com/researchspace/model/inventory/field/InventoryTimeField.java

```diff
@@ -38,9 +33,26 @@ public InventoryTimeField(String name) {
 
 	@Override
 	public boolean isSuggestedFieldForData(String data) {
+		return isValidTimeFormat(data);
+	}
+
+	@Override
+	public ErrorList validate(String fieldData){
+		ErrorList errors = super.validate(fieldData);
+		if(timeHasContentAndIsInvalid(fieldData)){
+				errors.addErrorMsg(String.format("%s is an invalid 24hour time format. Valid format is 00:00.", fieldData));
```

Comment:
Shouldn't that message say valid format is `HH:mm`, rather than `00:00`? That'd be consistent with date format message.